### ISSUE TITLE
define eslint config in .eslintrc file

### ISSUE THREAD

**Issue by ronaldruzicka** [actum/gulp-dev-stack#162]
I would like to have eslint rules defined in `.eslintrc` file like `.stylelintrc`, so that it's easily found and customized. It would also be good for [prettier plugin](https://github.com/prettier/prettier) available for Atom, Sublime Text and also Visual Studio/Code

**Comment by vbulant** [comment]
Let’s do that.

**Reasoning:**
While it seemed to be a noble idea to have _eslint-config-actum_, we’ve been running into multiple issues with it lately. Eg. people didn’t know how to extend it so they turned it out altogether and added .eslintrc anyway and on top of that they messed up dependencies so the whole pipeline didn’t work. On top of that, _prettier_ (and possibly other tools as well) needs .eslintrc to work properly.

To sum it up, it’s too much of an abstraction, let’s keep it simple.

**Comment by kettanaito** [comment]
Please note that `.eslintrc` is [deprecated](http://eslint.org/docs/user-guide/configuring#configuration-file-formats). One of the recommended usage may be `.eslintrc.js`.

**Comment by vbulant** [comment]
@kettanaito thanks!

Let’s use [`.eslintrc.js`](http://eslint.org/docs/user-guide/configuring#configuration-file-formats) and try if prettier works with it. If not, let’s stick with `.eslint` and wait for the update.
